### PR TITLE
Increasing resources and setting limits on Camunda API Prod deployment, increasing cleanup window to allow for completion

### PR DESCRIPTION
**Pull request by MartyFox**
### Jira link (if applicable)
https://tools.hmcts.net/jira/browse/DTSPO-16734
https://tools.hmcts.net/jira/browse/DTSPO-16612

### Change description ###
Increasing resources and setting limits on Camunda API Prod deployment, increasing cleanup window to allow for completion

### Checklist

### CODE REVIEW

File: apps/camunda/camunda-api/prod.yaml [hunk 1]
```diff
@@ -5,13 +5,16 @@ metadata:
 spec:
   values:
     java:
-      memoryRequests: "1Gi"
+      memoryRequests: '2Gi'
+      memoryLimits: '4Gi'
+      cpuRequests: '1'
```
Comment (thomast1906) [inline apps/camunda/camunda-api/prod.yaml:10]: do we need cpuRequests/limits?

Comment (MartyFox) [reply apps/camunda/camunda-api/prod.yaml]: removed.

Comment (thomast1906) [reply apps/camunda/camunda-api/prod.yaml]: ah ok, its a copy of perftest

File: apps/camunda/camunda-api/prod.yaml [hunk 2]
```diff
@@ -5,13 +5,16 @@ metadata:
...
       environment:
         WA_AUTO_CONFIGURE_TASKS_ENABLED: false
         CAMUNDA_BATCH_HISTORY_TTL: "P30D"
         CAMUNDA_BATCH_OPERATIONS: "P30D"
         CAMUNDA_HISTORY_TTL: "P30D"
         CAMUNDA_HISTORY_JOBLOG__TTL: "P30D"
-        CAMUNDA_HISTORY_CLEANUP_START_TIME: "03:00+0100"
+        CAMUNDA_HISTORY_CLEANUP_START_TIME: "00:30+0100"
         CAMUNDA_HISTORY_CLEANUP_END_TIME: "06:00+0100"
         CAMUNDA_HISTORY_CLEANUP_STRATEGY: "removalTimeBased"
```
[the remark this answers is not in the document]
Comment (reespozzi) [reply apps/camunda/camunda-api/prod.yaml]: It takes up job executor resources, I can't remember if they said there's a default number set for this somewhere so we can see what using 4 leaves us with, but they suggested it goes back to 1

**Review by reespozzi — APPROVED** [review]

**Pull request merged**